Tests/BugsnagTests/BugsnagStackframeTest.m

```diff
@@ -272,16 +272,24 @@ - (void)testRealCallStackSymbols {
         XCTAssertNotNil(stackframe.symbolAddress);
         XCTAssertNil(stackframe.type);
         
-        XCTAssertTrue([callStackSymbols[idx] containsString:stackframe.method] ||
-                      // Sometimes we do a better job at symbolication (-:
-                      [callStackSymbols[idx] containsString:@"???"] ||
-                      // But sometimes the best we can do is not great
-                      [stackframe.method isEqualToString:@"<redacted>"] ||
-                      // callStackSymbols contains the wrong symbol name - "__copy_helper_block_e8_32s"
-                      // lldb agrees that the symbol should be "__RunTests_block_invoke_2"
-                      [stackframe.method isEqualToString:@"__RunTests_block_invoke_2"] ||
-                      // Internal test function
-                      [stackframe.method isEqualToString:@"RunTestsFromRunLoop"]);
+        BOOL isStackframeValid = [callStackSymbols[idx] containsString:stackframe.method] ||
+            // Sometimes we do a better job at symbolication (-:
+            [callStackSymbols[idx] containsString:@"???"] ||
+            // But sometimes the best we can do is not great
+            [stackframe.method isEqualToString:@"<redacted>"] ||
+            // callStackSymbols contains the wrong symbol name - "__copy_helper_block_e8_32s"
+            // lldb agrees that the symbol should be "__RunTests_block_invoke_2"
+            [stackframe.method isEqualToString:@"__RunTests_block_invoke_2"] ||
+            // Internal test function
+        [stackframe.method isEqualToString:@"RunTestsFromRunLoop"];
+
+        if (!isStackframeValid) {
+            NSLog(@"callStackSymbol[%lu]: %@", (unsigned long)idx, callStackSymbols[idx]);
+            NSLog(@"stackframe.method=%s", [stackframe.method cStringUsingEncoding:NSUTF8StringEncoding]);
+            NSLog(@"stackframe.machoFile=%s", [stackframe.machoFile cStringUsingEncoding:NSUTF8StringEncoding]);
+            NSLog(@"stackframe.frameAddress=0x%llx", stackframe.frameAddress.unsignedLongLongValue);
+        }
+        XCTAssertTrue(isStackframeValid);
```

Comment:
I took the opportunity to improve the failure diagnostics for the unit test that fails under Xcode 26.